### PR TITLE
164072314 Redesign dashboard header panel

### PR DESCRIPTION
**Pull request by davenjeru**
#### What does this PR do?
Redesigns the dashboard's header

#### Description of Task to be completed?
 - Add responsiveness to the panel header.
 - Modify the panel header to look like the Invision mockups.

#### How should this be manually tested?
1. Start the project and log in to the application.
2. View the dashboard. It should resemble the screenshots below.
3. Test the responsiveness by increasing or reducing the width of the browser window.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
* [#164072314](https://www.pivotaltracker.com/story/show/164072314)

#### Screenshots (If applicable)
**Before**
image

**After**
image
image

### CODE REVIEW

**Comment by davenjeru** [comment]
This PR introduces a new environment variable `REACT_APP_BACKEND_URL`, which is the backend URL of ESA's REST API. I found it difficult to develop a frontend that had lag due to a large amount of data being fetched from ESA's staging URL and decided to add the variable which during development, would point to my mock server which returns less data.

File: src/index.css [hunk 1]
```diff
@@ -207,5 +204,5 @@ span {
 }
 
 * {
-    font-family: 'Open Sans', sans-serif;
-}
\ No newline at end of file
+    font-family: 'Arial', sans-serif;
```
[the remark this answers is not in the document]
Comment (davenjeru) [reply src/index.css]: Okay...

**Comment by davenjeru** [comment]
@matthewacha I have added the DIN pro font faces.

File: src/index.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (davenjeru) [reply src/index.css]: The font wasn't even available on the application. I had to add the font files.

**Comment by 09arnold** [comment]
Make the whole "user-info-container" toggle the dropdown for sign out and not just the caret beside it

**Comment by dotNesh** [comment]
@davenjeru From the mockups, the Andela logo has the 'A' in the middle. Also, should it be rotating?

**Comment by davenjeru** [comment]
> @davenjeru From the mockups, the Andela logo has the 'A' in the middle. Also, should it be rotating?

@dotNesh, before, there was no "A" on the logo and it was rotating. I found no reason to change it because it was not specifically mentioned in the story and the rotation or lack thereof cannot be shown on a mock-up.
Are you suggesting that I should change it @dotNesh cc @09arnold

**Comment by davenjeru** [comment]
@09arnold The whole `user-info-container` can now toggle the signout dropdown on click.

@dotNesh I have added the "A" and removed the rotation animation.

**Comment by matthewacha** [comment]
Update images on the PR

**Comment by davenjeru** [comment]
@matthewacha The signout button has been redesigned with guidance from @09arnold.

**Pull request merged**